nakadi.sh

```diff
@@ -2,7 +2,7 @@
 
 function waitForNakadi() {
   echo "Waiting for Nakadi to start up"
-  while True; do
+  while :; do
```

Comment:
does not look like it is related

Reply:
That's true.  I've just spotted it in the log with the failing tests and was wondering if it could be somehow related.  It wasn't.